### ISSUE TITLE
[1.4.2-beta.7] npm error on Ctrl+C shutdown from start script

### ISSUE THREAD

**Issue by JeremySaks** [meteor/meteor#7869]
I'm seeing npm errors when shutting down a Meteor project with Ctrl+C locally from the command line, if the project was started from an npm script.

I get this on a fresh project install so it is easy to reproduce:

``` bash
meteor create shutdown
cd shutdown
npm install
meteor update --release 1.4.2-beta.7
npm run start // node v.6.3.0, or
meteor npm run start // node v.4.5.0
```

The project starts up and runs fine. But a Ctrl+C shutdown triggers the following:

``` bash
^C
npm ERR! Darwin 16.0.0
npm ERR! argv "/Users/jeremysaks/.meteor/packages/meteor-tool/.1.4.2-beta.7.smk02e++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node" "/Users/jeremysaks/.meteor/packages/meteor-tool/.1.4.2-beta.7.smk02e++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "run" "start"
npm ERR! node v4.5.0
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! shutdown@ start: `meteor run`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the shutdown@ start script 'meteor run'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
```

It's true for `npm run start` and `meteor npm run start`. If I start the project directly using the `meteor` command, it doesn't occur.

**Comment by abernix** [comment]
What operating system are you on?  I actually can't reproduce this using your instructions no matter which version of Meteor/Node/NPM I use.  Can someone provide additional reproduction steps?  Either way, I'm not entirely sure there is much to be done about this on the Meteor side of things as the error is coming from NPM and this would mean that NPM was reacting differently to the signal.

If you add `|| true` after the `meteor run` command in the script, or alternatively, `npm run start -- silent`, it should avoid this error.  Does that fix it for you?

**Comment by JeremySaks** [comment]
I was able to eliminate this by simply rebuilding my .meteor installation folder. While I don't ultimately know the cause, given there's an easy workaround and it may not be Meteor specific, it's probably safe to close the issue.

For future readers coming across this, I'm on macOS Sierra and I encountered the error both on `npm run start` and `yarn run start` (with slightly different errors reported). It's conceivable yarn changes to the dependencies caused the issues, but I haven't taken the time to really explore it.

**Comment by abernix** [comment]
I'm going to close this unless we can accurately reproduce it.  It could be that for some reason there was legitimately an error, because I tried under what would seem to be the same circumstances.  I did dig through the NPM change-logs pretty extensively looking for _some_ reason why it might have began, but couldn't find a reason.

Thanks very much for reporting this though! It's good to have an issue at least searchable in the system for others to relate to. :smile:
